packages/maskbook/src/social-network/ui.ts

```diff
@@ -105,6 +105,10 @@ export interface SocialNetworkUIInjections {
      * This function should inject UI into the Post box
      */
     injectPostBox(): void
+    /**
+     * This function should inject the setup prompt
+     */
+    injectSetupPrompt(): void
```

Comment:
IMO it should be the unmount style, `injectSetupPrompt(): () => void`

Reply:
ok